sql/postgresql/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Does it make sense to have a nested `.gitignore` file? Is it possible to merge needed items with root `.gitignore`?